concepticondata/conceptlists/Clark-2008-441.tsv

```diff
@@ -0,0 +1,442 @@
+Clark-2008-441-300	300	raw	1959	RAW
+Clark-2008-441-301	301	old	2113	OLD (USED)
+Clark-2008-441-302	302	new	1231	NEW
+Clark-2008-441-303	303	near	1942	NEAR
+Clark-2008-441-304	304	narrow	1267	NARROW
+Clark-2008-441-305	305	low	3348	LOW (POSITION OR HEIGHT)
+Clark-2008-441-306	306	light weight	1052	LIGHT (WEIGHT)
+Clark-2008-441-307	307	left	244	LEFT
+Clark-2008-441-308	308	late	477	LATE
+Clark-2008-441-309	309	inside	1606	INSIDE
+Clark-2008-441-310	310	hot	1286	HOT
+Clark-2008-441-311	311	high	1265	HIGH
+Clark-2008-441-312	312	heavy	1210	HEAVY
+Clark-2008-441-313	313	hard	1884	HARD
+Clark-2008-441-314	314	green	1425	GREEN
+Clark-2008-441-315	315	good	1035	GOOD
+Clark-2008-441-316	316	full	1429	FULL
+Clark-2008-441-317	317	fragrant	376	FRAGRANT
+Clark-2008-441-318	318	fishy		
+Clark-2008-441-319	319	far	1406	FAR
+Clark-2008-441-320	320	early	672	EARLY
+Clark-2008-441-321	321	dry	1398	DRY
+Clark-2008-441-322	322	dead	1625	DEAD
+Clark-2008-441-323	323	cool	2483	COLD (OF WEATHER)
+Clark-2008-441-324	324	cold	1287	COLD
+Clark-2008-441-325	325	blind	1653	BLIND
+Clark-2008-441-326	326	black	163	BLACK
+Clark-2008-441-327	327	bitter	887	BITTER
+Clark-2008-441-328	328	below	1485	BELOW OR UNDER
+Clark-2008-441-329	329	bad	1292	BAD
+Clark-2008-441-330	330	alive	1422	BE ALIVE
+Clark-2008-441-331	331	above	1741	ABOVE
+Clark-2008-441-332	332	yellow	1424	YELLOW
+Clark-2008-441-333	333	to wash	1453	WASH
+Clark-2008-441-334	334	to write	1672	WRITE
+Clark-2008-441-335	335	to walk	1443	WALK
+Clark-2008-441-336	336	to wait	1590	WAIT (FOR)
+Clark-2008-441-337	337	to tell	1711	TELL
+Clark-2008-441-338	338	to tear	1735	TEAR (SHRED)
+Clark-2008-441-339	339	to talk	2708	TALK
+Clark-2008-441-340	340	to swim	1439	SWIM
+Clark-2008-441-341	341	to stoke		
+Clark-2008-441-342	342	to stir	696	STIR
+Clark-2008-441-343	343	to steam	2067	STEAM (VERB)
+Clark-2008-441-344	344	to steal	713	STEAL
+Clark-2008-441-345	345	to stand	1442	STAND
+Clark-2008-441-346	346	to soak	2069	SOAK (VERB)
+Clark-2008-441-347	347	to smoke	1689	SMOKE (INHALE)
+Clark-2008-441-348	348	to smell	2124	SMELL
+Clark-2008-441-349	349	to sleep	1585	SLEEP
+Clark-2008-441-350	350	to slash		
+Clark-2008-441-351	351	to sit	1416	SIT
+Clark-2008-441-352	352	to set trap		
+Clark-2008-441-353	353	to sell	1571	SELL
+Clark-2008-441-354	354	to see	1409	SEE
+Clark-2008-441-355	355	to scrape	1767	SCRAPE
+Clark-2008-441-356	356	to scold	299	SCOLD
+Clark-2008-441-357	357	to run	1519	RUN
+Clark-2008-441-358	358	to roast	1962	ROAST (SOMETHING)
+Clark-2008-441-359	359	to roar	2876	ROAR (OF SEA)
+Clark-2008-441-360	360	to ride horse	367	RIDE
+Clark-2008-441-361	361	to return	581	COME BACK
+Clark-2008-441-362	362	to rest	168	REST
+Clark-2008-441-363	363	to remember	707	REMEMBER
+Clark-2008-441-364	364	to pull gr		
+Clark-2008-441-365	365	to pull	1455	PULL
+Clark-2008-441-366	366	to pat		
+Clark-2008-441-367	367	to open mr		
+Clark-2008-441-368	368	to mow	190	MOW
+Clark-2008-441-369	369	to marry w		
```

Comment:
We have multiple marry concepts:
203	MARRY / 556 MARRY OFF / 2163 MARRY (AS WOMAN) / 2164 MARRY (AS MAN)
Maybe one of them fits.